schemas/reconciliation-query.json

```diff
@@ -1,77 +1,93 @@
 {
-    "$schema": "http://json-schema.org/schema#",
-    "$id": "https://reconciliation-api.github.io/specs/schemas/reconciliation-query.json",
-    "type": "object",
-    "description": "This schema validates a single reconciliation query",
-    "properties": {
-        "query": {
-            "type": "string",
-            "description": "A string to be matched against the name of the entities"
-        },
-        "properties": {
-            "type": "array",
-            "description": "An optional list of property mappings to refine the query",
-            "items": {
-                "type": "object",
-                "properties": {
-                    "pid": {
-                        "type": "string",
-                        "description": "The identifier of the property, whose values will be compared to the values supplied"
-                    },
-                    "v": {
-                        "description": "A value to match against the property values associated with the property on each candidate",
-                        "oneOf": [
-                            {
-                                "type": "string"
-                            },
-                            {
-                                "type": "number"
-                            },
-                            {
-                                "type": "boolean"
+    "$schema":"http://json-schema.org/schema#",
+    "$id":"https://reconciliation-api.github.io/specs/schemas/reconciliation-query.json",
+    "type":"object",
+    "description":"This schema validates a single reconciliation query",
```

Comment:
The description should also be updated, I think

Reply:
And perhaps the schema should be renamed to something like `reconciliation-query-batch.json` to avoid ambiguities.

Reply:
For now I won't be renaming the file. If there is only one type of query ("Multiple Query Mode" in the OR docs), I don't think we have to be more specific. IMO, it makes only sense if there also is another kind of query.

Regarding the description, I would just change it to "This schema validates a reconciliation query". Similar to the above, if there is only one query mode, this is totally fine I think. Ok?

Reply:
I don't know about you, but I would say that `{"query":"foo","type":"bar"}` is a JSON object that represents a "reconciliation query" (or would you use another term?)

The original docs do say "Multiple Query Mode": so it does mean that an object `{"q0":... ,"q1":...}` contains multiple queries. So an object `{"q0":..., "q1",…}` is not a query itself, it is a group of queries, or query batch, or query set…